src/plugins/intel_cpu/thirdparty/CMakeLists.txt

```diff
@@ -59,6 +59,17 @@ function(ie_add_onednn)
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -diag-disable=10121")
     endif()
 
+    # WA for old TBBConfig.cmake like tbb2019_20180718oss
+    # they don't check that imported target is already created
+    if(TBB_FOUND AND TBB_VERSION VERSION_GREATER_EQUAL 2018 AND TBB_VERSION VERSION_LESS_EQUAL 2019.4)
+        set(Threading_cmake_included ON)
+        set(DNNL_CPU_THREADING_RUNTIME "${THREADING}")
+        function(find_package_tbb)
+            # dummy
+        endfunction()
+        link_libraries(TBB::tbb)
```

Comment:
why do we need this? which variable is not filled?

Reply:
Because of the issue mentioned [here](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/CMakeLists.txt#L29-L30), we set `Threading_cmake_included` variable to ON, which makes [oneDNN to skip looking for TBB package](https://github.com/openvinotoolkit/oneDNN/blob/2a749c577f8a841a396d4bd46eaf311b7e7dc089/cmake/Threading.cmake#L20-L22).
So `link_libraries(TBB::tbb)` used here is yet another workaround to force targets in onednn directory to link with TBB.